conf/incliva.config

```diff
@@ -0,0 +1,52 @@
+// Profile details
+params {
+    config_profile_description = 'INCLIVA Health Research Institute profile for nf-core.'
+    config_profile_contact = 'Sheila Zúñiga Trejos - bioinfo@incliva.es'
+    config_profile_url = 'https://www.incliva.es/en/services/platforms/bioinformatics-unit/'
+}
+
+// Function to get hostname
+def getHostname() {
+    def process = 'hostname'.execute()
+    process.waitFor()
+    return process.text.trim()
+}
+
+// Function to set singularity path according to which host nextflow is running on
+def setHostConfig(String hostname) {
+    if (hostname == 'vlinuxcervantes3srv') {
+        System.out.println("\nINFO: working on ${hostname}\n")
+
+        // Resources details
+        params.max_memory = 60.GB
+        params.max_cpus = 15
+        singularity.cacheDir =  "/nfs/home/software/singularity/nf_cacheDir"
```

Comment:
This is the same for both, is it necessary to repeat?

Reply:
As mentioned in the previous comment, as I'm leaving I thought it might be better to leave it written explicitly for each host since they may need to change that path in the future.

Reply:
OK reasonable